content/blog/how-to-use-an-api-access-token-for-hpe-greenlake-for-compute-ops-management.md

```diff
@@ -0,0 +1,136 @@
+---
+title: How to use an API Access Token for HPE GreenLake for Compute Ops Management
+date: 2022-10-27T17:16:38.426Z
+author: Nisha Thomas
+authorimage: /img/nishathomas-small_blog.jpg
+disable: false
+tags:
+  - hpe-greenlake, hpe-greenlake-for-compute-ops-management
+---
+The Compute Ops Management REST API uses the OAuth 2.0 HPE GreenLake authentication flow. A limited lifetime access token needs to be provided in the header of each REST API request as the authorization bearer. The access token is associated with a subject (person or service) and has all the same permissions and privileges as the subject. The access token expires after 120 minutes.
```

Comment:
It would be helpful to add a little more of a benefit statement to this introduction and to not get quite so detailed so early in the post. You want to draw the reader in. Maybe it could be stated something like this:

Common identity frameworks and protocols use token-based strategies to secure access to applications and resources. OAuth 2.0 is one of the most popular, using access tokens and refresh tokens to allow an application to access resources hosted by other servers on behalf of a user. The Compute Ops Management REST API uses the OAuth 2.0 HPEGreenLake authentication flow, where a limited lifetime access token is provided in the header of each REST API request as the authorization bearer. The access token is associated with a subject (person or service) and retains all the same permissions and privileges as the subject. In this blog post, I will discuss the essential steps required to generate this access token.

(I'd put the part about the access token expiring a little lower somewhere in the copy)